src/framework/genesis_framework.gemspec

```diff
@@ -8,11 +8,12 @@ Gem::Specification.new do |gem|
   gem.email = 'opensourcesoftware@tumblr.com'
 
   gem.date = '2016-01-13'
-  gem.version = '0.6.8'
+  gem.version = '0.6.10'
   gem.add_dependency('genesis_promptcli', '~> 0.2', '>= 0.2.0')
   gem.add_dependency('genesis_retryingfetcher', '~> 0.4', '>= 0.4.0')
   gem.add_dependency('collins_client', '~> 0.2', '>= 0.2.0')
   gem.add_dependency('facter', '~> 2.0', '>= 2.0.0')
+  gem.add_dependency('activesupport', '5.1.1')
```

Comment:
Nah, if it can be loosened up thats ideal, unless there's a real specific bug we are avoiding (which we can assume there isn't because there is no comment next to the dependency)